tmt/utils.py

```diff
@@ -4618,6 +4618,9 @@ def generate_runs(
         # in abs_path to be generated.
         invalid_id = id_ and str(abs_child_path) != id_
         invalid_run = not abs_child_path.joinpath('run.yaml').exists()
+        if invalid_run and abs_child_path.name.startswith('run-'):
+            os.system(f"touch {abs_child_path.joinpath('run.yaml')}")
+            invalid_run = False
```

Comment:
I'm not sure I follow what's happening and why this helps. The commit message does very little in terms of explaining what's changing and especially why it needs to change. If a workdir lacks `run.yaml`, what help would it be to create a dummy, empty one? This is unclear from the PR. Can you shed more light on why a change like this solves the problem?

Reply:
> I didn't realize run.yaml will be used for cleaning guests😅, and I am
working on having try create run.yaml approach,this mr will fix the problem
of removing runs from tmt try,
as run.yaml will not be used for clean runs, so a dummy one works:)

Isn't it changing the way of discovering workdir for every user of this function? Let's say that for cleaning guests, `run.yaml` is not important and may not exist, but what about others? This function gets called from `Status.show()`, and it starts returning workdirs that were considered invalid in the past, which is not related to "cleaning guests" use case.

Also, if `run.yaml` will not be used for clean runs, does it need to exist? If we add a flag to `generate_runs()` to include also workdirs without `run.yaml`, does it have to be created? That might fool other parts of tmt into thinking this workdir is suddenly having apparently valid `run.yaml`, and therefore it's a valid workdir and should be inspected...